_posts/2016-04-09-linux-mint-razer-blade.markdown

```diff
@@ -0,0 +1,97 @@
+
+Use [this tutorial](https://community.linuxmint.com/tutorial/view/744) to mount the downloaded `.iso` to a USB drive thats at least 16GB.
+
+###Chapter 2: Install
+Plug the Razer Blade into a power source. Follow the directions in the user guide to boot from USB and start the install process.
+
+If you end up on the partition menu to customize the size of the install, make sure to add a few GB partition as "swap area". Among other things, this is important for resuming from a suspend. Learned this the hard way.
+
+Follow the rest of the directions to install completely and remove the USB drive (it'll prompt you).
+
+After install, boot up the machine, sign in, and install any updates from the Update Manager.
+
+###Chapter 3: Fixing hardware issues
+There are several parts of the hardware that just aren't hooked up right with the base install. I ran into issues with suspend black-screening indefinitely, two-finger scroll not working and other touchpad issues, and the internal webcam not working.
+
+The only concrete and consistent fix I've found is to change the version of the Linux kernel thats being used. This is a critical part of the system, so use caution, but I was able to fix most of my hardware issues this way.
+
+**Changing the kernel**
+
+Using the same Update Manager, navigate to View > Linux Kernels. This menu shows the full list of kernels available to switch to. As of this writing, the recommended and default kernel for Mint is **3.19.0-32**. I tried several different versions, some fixing some issues and creating others. The version I found most stable is **4.2.0-18**, and that's what I landed on.
+
+Load up 4.2.0-18 and then restart the machine. Mint should automatically boot into the latest version.
+
+This should fix the touchpad and suspend issues. The internal webcam issue still haunts me. I've worked around it by booting into Windows (works fine there) or using an external one. There are [several](https://insider.razerzone.com/index.php?threads/linux-on-a-new-blade.4115/page-4) [threads](https://bugs.launchpad.net/ubuntu/+source/cheese/+bug/1295247) that have mentioned this is as a common issue.
+
+###Chapter 4: Fixing software issues
+As I'll discuss in the last chapter, battery length a bit disappointing at first. With Cinnamon, by default I was barely getting 1.5 hours of battery life with general usage (internet browsing, videos, code editor)! Maybe 2.5 using Windows. At first this felt like a deal breaker, until I fixed the issues actually causing the power drain. Now I can get around 3.5 - 4. Which, in my opinion, is decent for a laptop gaming rig.
+
+**Switching to Nvidia drivers**
+
+At some point, it may be explicitly beneficial for you to switch to the Nvidia video card drivers over the default open-source XOrg drivers. Switching will also provide more fine grain control over the video card and provides the ability to switch between Performance settings and Power Saving settings.
```

Comment:
I believe it's either X.Org (how they actually stylize it) or everywhere else it's just Xorg